File: PhysTasks/Task3.py
import math

g = 9.81

H = float(input("Введите максимальную высоту подъема H (м): "))
L = float(input("Введите дальность полета L (м): "))

alpha_rad = math.atan((4 * H) / L)
alpha = math.degrees(alpha_rad)  

V = math.sqrt((g * L) / math.sin(2 * alpha_rad))

print(f"Угол α: {alpha:.2f} градусов")
print(f"Начальная скорость V: {V:.2f} м/с")

File: PhysTasks/Task3.cpp
#include <iostream>
#include <cmath>

using namespace std;

int main() {
    const double g = 9.81; 

    double H, L;

    cout << "Введите максимальную высоту подъема H (м): ";
    cin >> H;
    cout << "Введите дальность полета L (м): ";
    cin >> L;

    double alpha_rad = atan((4 * H) / L);
    double alpha = alpha_rad * 180.0 / M_PI; 

    double V = sqrt((g * L) / sin(2 * alpha_rad));

    cout << "Угол α: " << alpha << " градусов" << endl;
    cout << "Начальная скорость V: " << V << " м/с" << endl;

    return 0;
}
